docker-compose-dev.yaml

```diff
@@ -91,6 +91,32 @@ services:
       retries: 3
       start_period: 20s
 
+  offchain-consensus:
+    image: powerloom-offchain-consensus
+    profiles: ["consensus"]
+    expose:
+      - 9030
+    ports:
+      - 9030:9030
+    environment:
+      - UUID=$UUID
+    extra_hosts:
```

Comment:
@SwaroopH Noticed this issue of audit protocol starting before `offchain-consensus` and then throwing errors. 
I think it would be better to add health checks here and then add `offchain-consensus` to `audit-protocol` `depends_on` config. Not sure how that will work with profiles though.

Health check config
```
    healthcheck:
      test: ["CMD", "wget", "-qO-", "http://localhost:9030/currentEpoch"]
      interval: 10s
      timeout: 5s
      retries: 3
      start_period: 20s
```

Reply:
Yeah, I was trying to work around this - did not find the best solution.

But AP has a delay. There's no way the processes started before offchain-consensus, right? The only error is the epoch not being ready which is fine for the first start. Both AP and Pooler retry accordingly and don't fail healthchecks. Unless I am missing something.

Reply:
In the current config, project registration fails after 2 attempts and then everything exits.

Reply:
I just tested this on a fresh setup and it registers just fine. Even tried with all pairs. Can you test by resetting (last line) the volumes to make sure nothing funky is happening on your current setup?

Reply:
Resetting seems to have fixed the issue, this is very weird.